dockerfiles/Dockerfile.test-e2e-env

```diff
@@ -13,5 +13,6 @@ ENV     DISABLE_WARN_OUTSIDE_CONTAINER=1
 WORKDIR /work
 COPY    scripts/test/e2e scripts/test/e2e
 COPY    e2e/compose-env.yaml e2e/compose-env.yaml
+COPY    e2e/compose-env.yaml.experimental e2e/compose-env.yaml.experimental
```

Comment:
```
compose-env-experimental.yaml
```

Reply:
I used that to be able to do the `file="${file}:${file}.experimental"` hack :stuck_out_tongue: